lib/commons/dom/focus-disabled.js

```diff
@@ -14,6 +14,23 @@ function focusDisabled(el) {
     return true;
   }
 
+  let parentNode = vNode.parent;
+  while (parentNode) {
```

Comment:
Looking at this more closely this will cause a pretty big performance problem. Since each child element of the `aria-hidden` element will use this function, each of them will walk up the entire parent tree of the page. If the aria-hidden is placed near the top of the tree, that is a lot of checking of nodes already checked.

We should cache on the VirtualNode information about this lookup so each element only needs to go up a few nodes until it finds an element that has already gone through this lookup chain.

We'll want to do something similar to [is-hidden](https://github.com/dequelabs/axe-core/blob/develop/lib/core/utils/is-hidden.js) and cache the results of the recursive lookup onto each node we look at so we only ever have to look at a parent tree once.